### ISSUE TITLE
Publishing packages to `main` as well?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/cloudml-hypertune-feedstock#1]
IIUC [uploading unstable releases to the `main` label is permitted for packages with no stable releases]( https://github.com/conda-forge/cfep/blob/b561fa3d8a45cf0dcd6d9f8085cd42067301ef93/cfep-05.md#exceptions ). This was done for `black` originally as it had several beta releases. Given `cloudml-hypertune` is in a similar situation, am wondering if we should do the same here.

Asked this question in this thread ( https://github.com/conda-forge/staged-recipes/pull/15885#discussion_r687059941 ), but thought maybe it is better asked in a new issue.

**Comment by jakirkham** [comment]
cc @conda-forge/core (in case others have thoughts here)

**Comment by tarrade** [comment]
I am trying to push Vertex AI PM to provide a stable version with support for recent version of python (only for python 2.7 and 3.5). Let see if we can be a stable version after 3 years without any update (which make it a quite stable release)

**Comment by hmaarrfk** [comment]
Is this code still relevant in 2023? 

It seems the master branch has not had any activity since 2019. https://github.com/GoogleCloudPlatform/cloudml-hypertune/blob/master/hypertune/hypertune.py

for a Machine Learning project this seems "really old". 

That said, i feel like releasing the `.dev6` is Ok..... given that it is "stable" in the sense that it hasn't changed in 3 years.

**Comment by hmaarrfk** [comment]
and already released on pypi, albeit with a prerelease flag
https://pypi.org/project/cloudml-hypertune/#history

**Comment by tarrade** [comment]
yes still valid for GCP for hyper parameter tuning (no other alternative integrated with Vertex AI). I agree that GCP should reactivate and test that it is running with newer version of python >3.5

**Comment by tarrade** [comment]
I discussed with the PM. They will update it to support python 3.9 and python 3.10 but it will still be a dev release. Later this year the will integrated it to the Vertex AI SDK which already exist, have stable release
